bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/HBaseResultToMutationFn.java

```diff
@@ -43,7 +44,8 @@
  * A {@link DoFn} function that converts a {@link Result} in the pipeline input to a {@link
  * Mutation} for output.
  */
-class HBaseResultToMutationFn extends DoFn<KV<ImmutableBytesWritable, Result>, Mutation> {
+@InternalApi
+public class HBaseResultToMutationFn extends DoFn<KV<ImmutableBytesWritable, Result>, Mutation> {
```

Comment:
Should we move this class up a level?

Reply:
It could be in the beam package as it is now shared.

However I think we have some refactoring expected in near future for sequencefiles import and for this PR I was trying to avoid modifying too many sequencefiles related files so to keep the review simple.

Can we leave this to the refactoring when we are unifying the styles of these two pipeline?